static/to_compile/js/solution_map.ts

```diff
@@ -31,144 +28,93 @@ export class SolutionMap {
     this.#location = location
     this.#controller = controller
 
-    this.map = L.map(selector, {
-      preferCanvas: true,
-      zoomControl: false,
-    })
-
-    this.map.setView(DEFAULT_LOCATION, DEFAULT_ZOOM)
-    L.tileLayer("https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png", {
+    this.map = new Map({
+      container: selector,
+      style: {
+        version: 8,
+        sources: {
+          "carto-light": {
+            type: "raster",
+            tiles: [
+              "https://a.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png",
+              "https://b.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png",
+              "https://c.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png",
+            ],
+            tileSize: 256,
+            attribution: "© OpenStreetMap contributors, © CARTO",
+          },
+        },
+        layers: [
+          {
+            id: "carto-light-layer",
+            type: "raster",
+            source: "carto-light",
+            minzoom: 0,
+            maxzoom: 19,
```

Comment:
J'imagine qu'on veut utiliser cette valeur ? 

```suggestion
            maxzoom: DEFAULT_MAX_ZOOM,
```